installation-wizard-page/src/App.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { message } from "antd";
+import axios from "axios";
```

Comment:
Personally I prefer to use a single quote `'`.
But exist different code convention.
https://github.com/airbnb/javascript
https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines
https://eslint.org/docs/rules/jsx-quotes
https://github.com/airbnb/javascript/tree/master/react#quotes
We need to choose one to use everywhere.

Reply:
You're right, I'll find appropriate code guide for us (with linter rules too) and will add it to all our react repos